src/components/calcite-pick-list/calcite-pick-list.tsx

```diff
@@ -117,6 +129,10 @@ export class CalcitePickList {
       selectedValues.delete(value);
     }
     this.lastSelectedItem = item;
+    this.calciteListChange.emit(selectedValues);
+    /**
+     * @deprecated TODO: remove in the next major version bump.
```

Comment:
I think this comment is no longer needed since you added `@deprecated` to the emitter.

Reply:
technically not needed either way. Just thought it would be nice to flag this line of code. Ok with keeping it?